pkg/validation/app.go

```diff
@@ -135,35 +135,21 @@ func (v *Validator) validateCatalog(ctx context.Context, cr v1alpha1.App) error
 }
 
 func (v *Validator) validateConfig(ctx context.Context, cr v1alpha1.App) error {
-	if key.IsManagedByFlux(cr, v.projectName) {
-		return nil
-	}
-
 	if key.AppConfigMapName(cr) != "" {
-		ns := key.AppConfigMapNamespace(cr)
-		if ns == "" {
-			return microerror.Maskf(validationError, namespaceNotFoundReasonTemplate, "configmap", key.AppConfigMapName(cr))
-		}
-
-		_, err := v.k8sClient.CoreV1().ConfigMaps(ns).Get(ctx, key.AppConfigMapName(cr), metav1.GetOptions{})
+		err := v.validateConfigMapExistence(ctx, key.AppConfigMapName(cr), key.AppConfigMapNamespace(cr), "configmap", cr)
```

Comment:
```suggestion
		err := v.validateConfigMapExists(ctx, key.AppConfigMapName(cr), key.AppConfigMapNamespace(cr), "configmap", cr)
```

Name is clear but I think it could be shorter. WDYT?

Reply:
Sure, looks good and it makes sense to make it shorter 👍